pkg/cli/cmd/topaz/startrun.go

```diff
@@ -163,8 +163,10 @@ func getVolumes(cfg *config.Loader) ([]string, error) {
 	})
 
 	volumes := []string{
-		fmt.Sprintf("%s:/config:ro", cc.GetTopazCfgDir()),        // manually attach the configuration folder
-		fmt.Sprintf("%s:/root/.policy:ro", dockerx.PolicyRoot()), // manually attache policy store
+		fmt.Sprintf("%s:/config:ro", cc.GetTopazCfgDir()), // manually attach the configuration folder
+	}
+	if dockerx.PolicyRoot() != "" {
+		volumes = append(volumes, fmt.Sprintf("%s:/root/.policy:ro", dockerx.PolicyRoot())) // manually attache policy store
```

Comment:
Spelling `attache` -> `attach`

I am using https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker to catch these